easybuild/easyconfigs/p/PSORTb/PSORTb-3.0.4_CC.patch

```diff
@@ -0,0 +1,13 @@
+#Subtitue hardcoded cc for gcc to prevent 'cc1: error: unrecognized command line option "-fstack-protector-strong"'
+#Author: Ruben van Dijk (University of Groningen)
+--- bio-tools-psort-all/bio-tools-psort-modhmm/Makefile	2016-08-29 09:40:30.000000000 +0200
++++ bio-tools-psort-all/bio-tools-psort-modhmm/Makefile	2016-08-29 09:40:14.000000000 +0200
+@@ -34,7 +34,7 @@
+ # These definitions are from config.sh (via /software/software/Perl/5.22.1-foss-2016a/lib/perl5/5.22.1/x86_64-linux-thread-multi/Config.pm).
+ # They may have been overridden via Makefile.PL or on the command line.
+ AR = ar
+-CC = cc
++CC = gcc
```

Comment:
@RvDijk it's better/safer to use this construct, which should result in inheriting the `$CC` value from the environment, and only use `gcc` if it's not defined:

```
CC ?= gcc
```